qt/widgets/instrumentview/src/InstrumentActor.cpp

```diff
@@ -540,12 +546,6 @@ void InstrumentActor::sumDetectorsRagged(const std::vector<size_t> &dets,
                                          std::vector<double> &x,
                                          std::vector<double> &y,
                                          size_t size) const {
-  if (dets.empty() || size == 0) {
```

Comment:
Is this check not needed here too?

Reply:
Never mind. The collapsed diff view meant I didn't see that the protection has been added to the caller.